### PR TITLE
Manipulate the order of the records #2000

### PR DESCRIPTION
**Pull request by TobiasNx**
See #2000

### CODE REVIEW

File: src/main/resources/alma/alma.fix [hunk 1]
```diff
@@ -23,6 +23,12 @@ end
 
 add_field("@context","http://lobid.org/resources/context.jsonld")
 
+# Set empty elements to manipulate the order winthin the record.
```
Comment (dr0i) [inline src/main/resources/alma/alma.fix:26]:
```suggestion
# Set empty elements to manipulate the order within the record.
```

**Review by dr0i — APPROVED** [review]
besides the typo  +1

**Comment by blackwinter** [comment]
What about the `id` field? Apparently, it's not moved to the top.

**Comment by TobiasNx** [comment]
> What about the `id` field? Apparently, it's not moved to the top.

Why did I miss that?? 
I don't know why this does not work.
The `id` element is created here:

https://github.com/hbz/lobid-resources/blob/888a22fe10318230eb29cee87f736a806772f000/src/main/resources/alma/fix/identifiers.fix#L3

But somehow it is the last element to be passed on. And still is even when I add an empty ìd` element at the beginning.

**Comment by blackwinter** [comment]
Oh, wait, maybe it's because `id` is a _special_ field in Metafix. I'll take a look next week.

**Comment by blackwinter** [comment]
It's not in Metafix (I was confusing it with `_id`), but in lobid-resources: `JsonLdEtikett` always puts `id` last.

https://github.com/hbz/lobid-resources/blob/master/src/main/java/org/lobid/resources/EtikettJson.java#L80-L82

**Comment by TobiasNx** [comment]
> It's not in Metafix (I was confusing it with `_id`), but in lobid-resources: `EtikettJson` always puts `id` last.
> 
> https://github.com/hbz/lobid-resources/blob/2c1d7fa315aee8cd37c0d76e1aa476006760ba42/src/main/java/org/lobid/resources/EtikettJson.java#L80-L82

@dr0i can you help here. Why does the EtikettJson put the id always last?

**Comment by TobiasNx** [comment]
+1

**Pull request merged**